src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java

```diff
@@ -148,10 +148,8 @@ private void sendNotificationToRespondent(CaseDetails caseDetails) {
                 );
             }
         }
-
```

Comment:
Remove this file if no changes are there

Reply:
i will remove this change